test/Microsoft.Framework.Configuration.Binder.Test/ConfigurationBinderTests.cs

```diff
@@ -157,6 +157,14 @@ public void ConsistentExceptionOnFailedBinding(Type type)
         }
 
         [Fact]
+        public void BinderDoesNotChokeOnIndexer()
+        {
+            var builder = new ConfigurationBuilder();
+            var config = builder.Build();
+            var options = ConfigurationBinder.Bind<List<string>>(config);
```

Comment:
This used to throw before the fix, so I don't typically use Assert.NotThrow... but that's basically what it this test is